Makefile

```diff
@@ -45,7 +45,7 @@ $(BUILD_DIR)/index.html.liquid: frontend/index.html frontend/index.js frontend/s
 $(all_html): $(BUILD_DIR)/%.html: $(BUILD_DIR)/%.html.liquid $(CONF) $(html_renderer)
 	node $(html_renderer) -c $(CONF)  -o $@ $<
 	@# remove indents to reduce size
-	sed -E -i 's/^\s+//g' $@
+	sed -E -i '' 's/^\s+//g' $@
```

Comment:
For FreeBSD sed, it is not recommended to use a zero-length extension, use a meaningful extension (like `.backup` instead.

Reply:
Fixed, I used perl instead, it's more portable.